frontend/src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -49,6 +55,29 @@ const StyledCardLink = styled(Link)(({ theme }) => ({
     pointer: 'cursor',
 }));
 
+const StyledButtonGroup = styled(ToggleButtonGroup)(({ theme }) => ({
+    'html[data-theme="light"] button': {
+        '--hover-background-color': '#615BC2',
+    },
+    'html[data-theme="dark"] button': {
+        '--hover-background-color': '#34325E',
+    },
```

Comment:
No, you're right, this doesn't work. We don't use `data-theme` in Unleash. But this is just a placeholder with the correct values for now. It should be removed once we find the right variable to use for the actual hover background color.

Reply:
Let's note it in code. `// FIXME: find hover background color and remove placeholder`?

Reply:
Couldn't you simply do `&:hover { }` and set it to a variable from the theme? Or are these values missing from the theme?

Reply:
Yeah, just using `:hover` should work, but no, I can't figure out what the values in the theme are. Do you know?